packages/nx/src/command-line/release/config/filter-release-groups.spec.ts

```diff
@@ -193,9 +175,22 @@ describe('filterReleaseGroups()', () => {
             "resolvedVersionPlans": false,
             "version": {
               "conventionalCommits": false,
-              "generator": "",
-              "generatorOptions": {},
+              "currentVersionResolver": undefined,
```

Comment:
These values are not correct, the should not be `undefined`, they should be their default values to match how all other config construction has been set up to work

Reply:
Wait sorry I see this is actually because of this `createVersionConfig` abstraction, rather than the config logic